Tests/SwiftLintFrameworkTests/ImplicitlyUnwrappedOptionalRuleTests.swift

```diff
@@ -23,4 +23,28 @@ class ImplicitlyUnwrappedOptionalRuleTests: SwiftLintTestCase {
         verifyRule(description, ruleConfiguration: ["mode": "all"],
                    commentDoesntViolate: true, stringDoesntViolate: true)
     }
+
+    func testImplicitlyUnwrappedOptionalRuleWarnsOnOutletsInWeakMode() {
+        let baseDescription = ImplicitlyUnwrappedOptionalRule.description
+        let triggeringExamples = [
+            Example("private weak var label: UILabel!"),
+            Example("weak var label: UILabel!"),
+            Example("@objc weak var label: UILabel!")
```

Comment:
Please add violation markers (`↓`) to these examples so we also test where the rule triggers.